api/openid_config.json.example

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you change the file name to `openid_config.example.json`, so that editors can syntax highlight properly?